### ISSUE TITLE
train

### ISSUE THREAD

**Issue by YAN287-hub** [zhengchen1999/DAT#75]
<html>
<body>

45K | 30.6875
-- | --
50K | 30.7549
65K | 30.6819
80K | 30.5622
85K | 30.3650
95K | 29.9670
105K | 29.8656
115K | 30.0508
，这是我现在遇到的训练问题为什么后面越训练指标越低呢，我现在再跑对比实验，训练的是单通道红外数据集，改完之后，# general settings
name: train_DAT_DroneVehicle_x4
model_type: DATModel
scale: 4
num_gpu: 1
manual_seed: 0

# dataset and data loader settings
datasets:
  train:
    task: SR
    name: DroneVehicle_train
    type: PairedImageDataset
    dataroot_gt: datasets/DroneVehicle/trainsets/HR
    dataroot_lq: datasets/DroneVehicle/trainsets/LRx4
    filename_tmpl: '{}'
    io_backend:
      type: disk

    gt_size: 256
    use_hflip: true
    use_rot: true

    # data loader
    use_shuffle: True
    num_worker_per_gpu: 12
    batch_size_per_gpu: 4
    dataset_enlarge_ratio: 1
    prefetch_mode: ~

  val:
    task: SR
    name: DroneVehicle_val
    type: PairedImageDataset
    dataroot_gt: datasets/DroneVehicle/valsets/HR
    dataroot_lq: datasets/DroneVehicle/valsets/LRx4
    filename_tmpl: '{}'
    io_backend:
      type: disk

# network structures
network_g:
  type: DAT
  upscale: 4
  in_chans: 1
  img_size: 64
  img_range: 1.
  split_size: [8,32]
  depth: [6,6,6,6,6,6]
  embed_dim: 180
  num_heads: [6,6,6,6,6,6]
  expansion_factor: 4
  resi_connection: '1conv'

# path
path:
  pretrain_network_g: ~
  strict_load_g: true
  resume_state: ~

# training settings
train:
  optim_g:
    type: Adam
    # lr: !!float 2e-4
    lr: !!float 2e-4
    weight_decay: 0
    betas: [0.9, 0.99]

  scheduler:
    type: MultiStepLR
    # milestones: [ 250000, 400000, 450000, 475000 ]
    milestones: [ 100000 ]
    gamma: 0.5

  # total_iter: 500000
  total_iter: 200000
  warmup_iter: -1  # no warm up

  # losses
  pixel_opt:
    type: L1Loss
    loss_weight: 1.0
    reduction: mean

# validation settings
val:
  val_freq: !!float 5e3
  save_img: False

  metrics:
    psnr: # metric name, can be arbitrary
      type: calculate_psnr
      crop_border: 4
      test_y_channel: False
    ssim:
      type: calculate_ssim
      crop_border: 4
      test_y_channel: false

# logging settings
logger:
  print_freq: 200
  save_checkpoint_freq: !!float 5e3
  use_tb_logger: True
  wandb:
    project: ~
    resume_id: ~

# dist training settings
dist_params:
  backend: nccl
  port: 29500
这是我的训练配置文件，我在里面加了一个  ema_decay: 0.999，是因为这个原因吗，请博主回答一下我的问题谢谢，

<img width="686" height="539" alt="Image" src="https://github.com/user-attachments/assets/4e15b50a-2434-406f-aec9-5333991bbe7b" />
